.github/workflows/test.yml

```diff
@@ -18,30 +18,28 @@ jobs:
       - uses: pre-commit/action@v3.0.0
 
       - name: install
+        shell: zsh -i -euo pipefail {0}
```

Comment:
zsh isn't really a scripting language -- we should still be using bash

Reply:
people are gonna be using zsh on macos though, but i guess we can switch since in the future i'd like this to work for people on linux as well